python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -292,7 +297,11 @@ def current_processor_usage(self) -> ExecutionResources:
         )
 
     def pending_processor_usage(self) -> ExecutionResources:
-        num_pending_workers = self._actor_pool.num_pending_actors()
+        # Both pending and restarting actors count towards pending processor usage
```

Comment:
We also need to report the number of actors of different states on the progress bar. 
Currently, it shows something like `Actors X, (pending Y)`.  where X and Y are reported by `PhysicalOperator.num_active_actors/num_pending_actors`. One problem is that the definition of active and pending doesn't align with what we have in `_ActorPool`
So I'm thinking of simplifying this to one single `actor_info_progress_str` method that reports something like `Actors X (alive A, restarting B, pending C).`, where X is the total number, and the parenthesis can be dropped when all actors are alive. 
cc @scottjlee

Reply:
corresponding code in `OpState.summary_str`

Reply:
agree with Hao on the proposed updated actor info string.

Reply:
Sure let me go through that API